app/pom.xml

```diff
@@ -26,6 +26,7 @@
     <maven.compiler.source>17</maven.compiler.source>
     <maven.compiler.target>17</maven.compiler.target>
     <start-class>com.ozonehis.eip.Application</start-class>
+    <liquibaseVersion>4.24.0</liquibaseVersion>
```

Comment:
Why not use the latest? `v4.27.0`

Reply:
Just to be consistent across services https://github.com/openmrs/openmrs-eip/blob/master/pom.xml#L33